x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/__mocks__/request_responses.ts

```diff
@@ -295,6 +338,13 @@ export const createActionResult = (): ActionResult => ({
   config: {},
 });
 
+export const nonRuleAlert = () => ({
+  ...getResult(),
+  id: '04128c15-0d1b-4716-a4c5-46997ac7f3bc',
+  name: 'Non-Rule Alert',
+  alertTypeId: 'something',
+});
```

Comment:
I think what you want here is to return the Type and remove the `id` since it does not add any extra value and you are trying to override the name and alertTypeId like so:

```ts
export const nonRuleAlert = (): RuleAlertType => ({
  ...getResult(),
  name: 'Non-Rule Alert',
  alertTypeId: 'something',
});
```

Reply:
@FrankHassanabad Defining an ID distinct from the one mocked for our Rule Alert was intentional. It's not exercised in the test, but I did it so that we don't risk a false-positive if we ever compared IDs somewhere.